nativelib/src/main/scala-3/scala/scalanative/runtime/Continuations.scala

```diff
@@ -0,0 +1,168 @@
+package scala.scalanative.runtime
+
+import scala.scalanative.unsafe.*
+import scala.scalanative.runtime.libc.{malloc, free}
+import scala.scalanative.runtime.Intrinsics.*
+import scala.collection.mutable
+
+import scala.util.Try
+import scala.scalanative.meta.LinktimeInfo.isWindows
+
+object Continuations:
+  import ContImpl.*
+
+  opaque type BoundaryLabel[-T] = ContImpl.BoundaryLabel
+
+  ContImpl.init(CFuncPtr2.fromScalaFunction(allocateBlob))
```

Comment:
Some additional info abut the init would be usefull, also it would be greate to explain more about these blobs